rust/test_service_elections/src/lib.rs

```diff
@@ -153,7 +238,7 @@ mod service {
         );
 
         let table = CandidatesTable::open();
-        let mut idx = table.get_index::<ByElectionVotesCount>(1);
+        let mut idx = table.get_index_by_election_votes_key();
```

Comment:
The idx is typed for the user, since we will auto generate the method from the macro. So I believe we are ok now.